### DISCUSSION TITLE
Group icon resource names

### DISCUSSION THREAD

**Discussion by johnmcd88** [osmandapp/OsmAnd#18986] [General]
Hi,

Is there some where I can find a list of the resource names for each of the group icons? For example, if I assign the fire station icon to a group and export my favorites, the resulting gpx file has a group item with a qualifier of 'icon="amenity_fire_station"'. I'm trying to create a gpx file that contains a list of pre-defined groups with names, colors, icons and backgrounds that my hiking group can import so that when they create a new favorite they're all selecting from a standardized set of groups.

If the answer is 'look in the source code' on github could you provide a specific module name? I've programmed everything from mainframes to embedded systems but I've never touched Android programming, so I'm not super familiar with naming/coding conventions.

Thanks

John

**Comment by dmpr0** [comment]
Hi!
You can find list of all icons available for Favourites or track Waypoints here https://github.com/osmandapp/OsmAnd-resources/blob/master/poi/poi_categories.json

**Reply by johnmcd88** [reply]
Excellent - thank you!
